Client/Frontend/Browser/BackForwardTableViewCell.swift

```diff
@@ -50,6 +50,14 @@ class BackForwardTableViewCell: UITableViewCell {
         didSet {
             if let s = site {
                 faviconView.setFavicon(forSite: s, onCompletion: { [weak self] (color, url) in
+                    
+                    // Mozilla Firefox Start Page
+                    let urlString = (url?.isLocal)! ? url?.getQuery()["url"]?.unescape() : url?.absoluteString
+                    if urlString == nil {
```

Comment:
Similar story as above. There are rarely times you'll need to check for nil values like this. Opt for using guard/ifs.